src/components/atoms/charts/ContribRadar/index.jsx

```diff
@@ -0,0 +1,187 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> Additional
+// Charts for displaying user contribution distribution (Chart.js 2)
+import { Radar } from "react-chartjs-2";
+//#endregion
+
+//#region > Components
+/**
+ * @class A contribution statistic which displays the the ratio of
+ *        commits, issues, pull requests and code reviews.
+ */
+class ContribRadar extends React.Component {
+  state = {
+    dataRadarOptions: {
+      responsive: true,
+      elements: {
+        line: {
+          tension: 0,
+        },
+      },
+      tooltips: {
+        enabled: false,
+      },
+      legend: {
+        display: false,
+      },
+      elements: {
+        point: {
+          radius: 0,
+        },
+      },
+      scale: {
+        pointLabels: {
+          fontSize: 11,
+          color: "grey",
+        },
+        gridLines: {
+          display: false,
+        },
+        ticks: {
+          display: false,
+        },
+      },
+      scales: {
+        yAxes: [
+          {
+            gridLines: {
+              display: false,
+              drawBorder: false,
+            },
+            ticks: {
+              display: false,
+            },
+          },
+        ],
+        xAxes: [
+          {
+            gridLines: {
+              display: false,
+              drawBorder: false,
+            },
+            ticks: {
+              beginAtZero: true,
+              display: false,
+            },
+          },
+        ],
+      },
+    },
+  };
+
+  componentDidMount = () => {
+    // Start of by rendering chart with current year
+    this.calculateSources(null);
+  };
+
+  componentWillReceiveProps(nextProps) {
+    // Update chart
+    this.calculateSources(nextProps.year);
+  }
+
+  fillChart = (results) => {
+    console.log(results);
+    
+    if (results) {
+      this.setState({
+        dataRadar: {
+          labels: [
+            `Code Review ${results[0].data[0]}%`,
+            `Issues ${results[0].data[1]}%`,
+            `Pull Request ${results[0].data[2]}%`,
+            `Commits ${results[0].data[3]}%   `,
+          ],
+          datasets: results,
+        },
+      });
+    } else {
+      this.setState({ dataRadar: null });
+    }
+  };
+
+  calculateSources = (nextPropsYear) => {
+    const { statistic } = this.props;
+    let totalReviews = 0,
+      totalIssues = 0,
```

Comment:
Please fix indentation.

Reply:
This is the default indentation provided by prettier.